gnomad/utils/constraint.py

```diff
@@ -577,6 +577,8 @@ def build_models(
     high_cov_definition: int = COVERAGE_CUTOFF,
     upper_cov_cutoff: Optional[int] = None,
     skip_coverage_model: bool = False,
+    coverage_metric: str = "exomes_AN_percent",
+    cov_model_type: str = "linear",
```

Comment:
Same with this, let's make `logrithmic` the default. Just since v2.1.1 constraint is still the one that has been more vetted, and then this function won't change it's behavior from what users might be expecting